doc/sphinx/source/context-operations.rst

```diff
@@ -111,8 +111,8 @@ If you configure a context for a logic such as ``"QF_NRA"`` or ``"QF_UFNIA"``
 then MCSat will be automatically selected.
 
 The MCSat solver does not currently support as many features as the
-DPLL(T) implementation. In particular, MCSat does not yet support push
-and pop.
+DPLL(T) implementation. In particular, MCSat does not yet support
+computing unsat cores.
```

Comment:
However, unsat cores in MCSat can be computed using c:function:: yices_check_context_with_model and c:function:: yices_get_model_interpolant.

Reply:
> However, unsat cores in MCSat can be computed using c:function:: yices_check_context_with_model and c:function:: yices_get_model_interpolant.

Done in cfcd6907